src/NuGet.Clients/NuGet.VisualStudio/NuGet.VisualStudio.nuspec

```diff
@@ -12,8 +12,8 @@
     <copyright>© Microsoft Corporation. All rights reserved.</copyright>
   </metadata>
   <files>
-     <file src="NuGet.VisualStudio.dll" target="lib\net45\NuGet.VisualStudio.dll" />
-     <file src="NuGet.VisualStudio.XML" target="lib\net45\NuGet.VisualStudio.XML" />
+     <file src="NuGet.VisualStudio.dll" target="lib\net46\NuGet.VisualStudio.dll" />
```

Comment:
Actually good point. Initially I planned to add new interface to the existing package. Now when it's placed in its own will revert these changes.